### PR TITLE
option to hide clock and date

### PR DESCRIPTION
**Pull request by jlambert121**
This card is awesome, added it to my tablet display.  I already have time and date already though where I want it so would like to hide it and replace the larger block with the current temperature.

image

### CODE REVIEW

**Comment by jlambert121** [comment]
I had the ternary operator backwards for the date and didn't notice it.  

image

**Comment by pkissling** [comment]
thanks for the PR. i'll have a look once i'm back from vacation in january

**Comment by jlambert121** [comment]
Thanks!  Sorry I couldn't push this over the line - I just haven't done any python before.  Enjoy your vacation!

File: README.md [hunk 1]
```diff
@@ -99,7 +99,9 @@ Depending on your Home Assistant's configuration, your weather provider might de
...
 | hide_today_section    | boolean          | **Optional** | Hides the cards today section (upper section), containing the large weather icon, clock and current date                                                                                               | `false`     |
-| hide_forecast_section | boolean          | **Optional** | Hides the cards forecast section (lower section),containing the weather forecast                                                                                                                       | `false`     |
+| hide_forecast_section | boolean          | **Optional** | Hides the cards forecast section (lower section), containing the weather forecast                                                                                                                      | `false`     |
+| hide_clock            | boolean          | **Optional** | Hides the clock from the today section                                                                                                                                                                 | `false`     |
```
Comment (pkissling) [inline README.md:103]: Hides the clock from the today section and prominently displays the current temperature instead.

**Review by pkissling — APPROVED** [review]
hey @jlambert121,
sorry for taking so long with the review. i'm afraid you have to rebase the PR and resolve some merge conflicts before being able to merge it... :(

i only added one suggestion in the PR regarding the feature description in the README.md. other than that the code looks good. i also tested it on my HA instance without any issue.

appreciate your work!

**Comment by pkissling** [comment]
Resolves #141

**Comment by pkissling** [comment]
> hey @jlambert121, sorry for taking so long with the review. i'm afraid you have to rebase the PR and resolve some merge conflicts before being able to merge it... :(
> 
> i only added one suggestion in the PR regarding the feature description in the README.md. other than that the code looks good. i also tested it on my HA instance without any issue.
> 
> appreciate your work!

i solved the merge conflict for you

File: README.md [hunk 2]
```diff
@@ -105,7 +105,9 @@ Depending on your Home Assistant's configuration, your weather provider might de
...
 | date_pattern          | string           | **Optional** | Pattern to use for time formatting. If not provided, falls back to the default date formatting of the configured language. See [date-fns](https://date-fns.org/v2.29.3/docs/format) for valid patterns | `P`         |
 | hide_today_section    | boolean          | **Optional** | Hides the cards today section (upper section), containing the large weather icon, clock and current date                                                                                               | `false`     |
-| hide_forecast_section | boolean          | **Optional** | Hides the cards forecast section (lower section),containing the weather forecast                                                                                                                       | `false`     |
+| hide_forecast_section | boolean          | **Optional** | Hides the cards forecast section (lower section), containing the weather forecast                                                                                                                      | `false`     |
```
Comment (pkissling) [inline README.md:108]: Please also add the properties to https://github.com/pkissling/clock-weather-card#full-configuration

**Review by pkissling — CHANGES_REQUESTED** [review]

**Comment by jlambert121** [comment]
readme updated!

**Review by pkissling — APPROVED** [review]

**Pull request merged**